storage/api/test/BucketAclCommandTest.php

```
@@ -0,0 +1,104 @@
+        $application->add(new BucketAclCommand());
+        $this->commandTester = new CommandTester($application->get('bucket-acl'));
+    }
+
+    public function testBucketAcl()
+    {
+        if (!self::$hasCredentials) {
+            $this->markTestSkipped('No application credentials were found.');
+        }
+        if (!$bucketName = getenv('GOOGLE_STORAGE_BUCKET')) {
+            $this->markTestSkipped('No storage bucket name.');
+        }
+
+        $this->commandTester->execute(
+            [
+                'bucket' => $bucketName,
+            ],
+            ['interactive' => false]
+        );
+
+        $this->expectOutputRegex("/: OWNER/");
+    }
+
+    public function testManageBucketAcl()
+    {
+        if (!self::$hasCredentials) {
+            $this->markTestSkipped('No application credentials were found.');
+        }
+        if (!$bucketName = getenv('GOOGLE_STORAGE_BUCKET')) {
+            $this->markTestSkipped('No storage bucket name.');
+        }
+
+        $this->commandTester->execute(
+            [
+                'bucket' => $bucketName,
+                '--entity' => 'allAuthenticatedUsers',
+                '--create' => true,
+            ],
+            ['interactive' => false]
+        );
+        $this->expectOutputRegex("/Added allAuthenticatedUsers (READER) to \S+ ACL/");
+
+        $this->commandTester->execute(
+            [
+                'bucket' => $bucketName,
+                '--entity' => 'allAuthenticatedUsers',
+            ],
+            ['interactive' => false]
+        );
+        $this->expectOutputRegex("/allAuthenticatedUsers: READER/");
+
+        $this->commandTester->execute(
+            [
+                'bucket' => $bucketName,
+                '--entity' => 'allAuthenticatedUsers',
+                '--delete' => true,
+            ],
+            ['interactive' => false]
+        );
+        $this->expectOutputRegex("/Deleted allAuthenticatedUsers from \S+ ACL/");
```

This test will pass when the API call silently fails. Ideally can we test the actual behavior? Maybe you can file a bug for it and tackle it later.